bytecomp/symtable.ml

```diff
@@ -348,9 +348,10 @@ let defined_globals patchlist =
     patchlist
 
 let required_globals patchlist =
+  let not_predef id = not (Ident.is_predef id) in
   List.fold_left (fun accu rel ->
       match rel with
-      | (Reloc_getglobal id, _pos) -> id :: accu
+      | (Reloc_getglobal id, _pos) when (not_predef id) -> id :: accu
```

Comment:
I have a slight preference for the following style:
```ocaml
| (Reloc_getglobal id, _pos) -> if Ident.is_predef id then accu else id :: accu
```
If you prefer your version, I'd still suggest to name your auxiliary function `is_comp_unit` instead of `not_predef`.